src/prevector.h

```diff
@@ -8,6 +8,7 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
+#include <assert.h>
```

Comment:
Does assert.h need to be included here? This doesn't seem to add any assertions.